test/ppspp_options_SUITE.erl

```diff
@@ -56,7 +57,7 @@ getters(_Config) ->
     Swarm_id = convert:hex_string_to_padded_binary(Root_Hash),
     Options = ppspp_options:use_default_options(Swarm_id),
     %% confirm that we can unwrap options from within a Datagram-style orddict
-    Options = ppspp_options:get_options([{foo, bar}, {fu, baa}, Options]),
+    Options = ppspp_options:get_options(#{foo => bar, fu => baa, options => Options}),
```

Comment:
Line 60 is too long:     Options = ppspp_options:get_options(#{foo => bar, fu => baa, options => Options}),.